pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -7,6 +7,8 @@
 , emacs
 , ruby
 , which, dtach, openssl, bash, gdb, man
+, withDocs ? true
```

Comment:
How about having a different output for docs instead of disabling it?

Reply:
I'm afraid I don't know what that means/how to do it ^_^

Reply:
You basically add:

```
outputs = [ "out" "doc" ];
```

Than it should move "$out/usr/share/doc" to "$doc" instead of "$out".
Having multiple outputs is a feature of nixpkgs and described in the nixpkgs manual here: https://nixos.org/nixpkgs/manual/#chap-multiple-output

Reply:
Something like this?